test/AlarmClockTest.cpp

```diff
@@ -48,13 +49,13 @@ TEST_F(AlarmClockTest, GetUsSleepTimeInUs) {
 TEST_F(AlarmClockTest, GetMsSleepTimeInUs) {
    int ms = 123456;
    AlarmClock<milliseconds> alerter(ms);
-   EXPECT_EQ(ConvertToMicroSeconds(milliseconds(ms)), alerter.SleepTimeUs());
+   EXPECT_EQ(ConvertToMicroSeconds(milliseconds(ms)), static_cast<unsigned int>(alerter.SleepTimeUs()));
```

Comment:
Consider modifying the test class to return signed int for ConvertToMicroSeconds.